src/urbit/mar/chess/archive.hoon

```diff
@@ -0,0 +1,36 @@
+/-  *chess
+/+  chess
+=,  format
+|_  archive=((mop @dau chess-game) gth)
```

Comment:
This mark is useful for scrying, though it's not what we need to meet the goals of the PR (but it's still valuable to keep). We need another mark, similar to `mar/chess/game.hoon`, but for archived games.

I think we should change how we expect people to scry for archived games, if they need to, just to make the code easier to deal with on our end. Scrying along `[%x %archive

- See above comment for moving the `mar/chess/game.hoon` mark to `mar/chess/game/active.hoon`
- Make a new mark in `mar/chess/game/archived.hoon`
- It should be almost identical, but instead of having the fields used by `ActiveGameInfo`, it has the fields used by `ArchivedGameInfo` (i.e. just `'result'`